docs/conf.py

```diff
@@ -43,6 +43,12 @@
     print('ERROR: the documentation requires the sphinx-astropy package to be installed')
     sys.exit(1)
 
+try:
+    import sphinx_changelog
```

Comment:
Unused import will trigger flake8 when we do make it stricter in the future, so might as well.
```suggestion
    import sphinx_changelog  # noqa
```